From 0f886f42c2a34e560b0ddb03b6ddc2873123bd00 Mon Sep 17 00:00:00 2001
From: Theodore Ts'o <tytso@mit.edu>
Date: Sun, 21 Jul 2019 13:13:24 -0400
Subject: [PATCH] e2scrub_all_cron: check to make sure e2scrub_all exists

Since e2scrub_all.cron is marked as a config file, it can hang around
after the package is removed, in which case e2scrub_all might not be
present.  So check to make sure e2scrub_all exists before trying to
execute it.

Addresses-Debian-Bug: #932622

Signed-off-by: Theodore Ts'o <tytso@mit.edu>
Reported-by: Laurent Bigonville <bigon@debian.org>
---
 scrub/e2scrub_all_cron.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scrub/e2scrub_all_cron.in b/scrub/e2scrub_all_cron.in
index f9cff878..fcfe415f 100644
--- a/scrub/e2scrub_all_cron.in
+++ b/scrub/e2scrub_all_cron.in
@@ -62,6 +62,7 @@ on_ac_power() {
 	return 0
 }
 
+test -e @root_sbindir@/e2scrub_all || exit 0
 test -e /run/systemd/system && exit 0
 on_ac_power || exit 0
 
-- 
2.22.0

